### PR TITLE
Adding events/actions to brid-player

### PR DESCRIPTION
**Pull request by grajzer**

### CODE REVIEW

**Comment by grajzer** [comment]
Hi @alanorozco , sorry for not specifying. 

This change was made so that users can use amp event system to listen to some of its ad-related events.

Also added destroy action that can be used to call destroy method on player element.

This is because some users wanted more control of elements on their AMP pages. For example, they wanted a possibility to hide player and show some other elements on page in case of ad error.

<amp-brid-player
  id="myVideo"
  data-partner="264"
  data-player="4144"
  data-video="13663"
  width="480" height="270"

  on="adError:myVideo.hide,myVideo.destroy,.........."
  >
</amp-brid-player>

**Comment by grajzer** [comment]
Thanks for the explanation @alanorozco .

Changed trust of events dispatched from player to low. They cannot be used to change layout of the pages, so I hope this is ok now.

**Comment by alanorozco** [comment]
@grajzer What does the `destroy` command do?

**Comment by grajzer** [comment]
Hi @alanorozco it enables users to destroy player element inside the iframe. Iframe element is not touched. Is that ok?

**Comment by alanorozco** [comment]
@grajzer What's the use case for destroy? How can it be triggered by other components?

I'm still somewhat confused since the code doesn't seem to match the behavior described :) 

https://github.com/ampproject/amphtml/blob/4f55148fcfabdaeff03493c953684551fdfe99b3/extensions/amp-brid-player/0.1/amp-brid-player.js#L326-L328

This is triggering an action, not an event. This would be like doing `<button on="tap:my-brid-player.adError">`. `adError`, like the other `BridEvents` is not a registered action.

File: extensions/amp-brid-player/0.1/amp-brid-player.js [hunk 1]
```diff
@@ -175,6 +183,9 @@ class AmpBridPlayer extends AMP.BaseElement {
 
     installVideoManagerForDoc(element);
     Services.videoManagerForDoc(element).register(this);
+
+    /** Register Brid actions */
+    this.registerDefaultAction(() => this.destroy(), BridActions.DESTROY);
```
Comment (alanorozco) [inline extensions/amp-brid-player/0.1/amp-brid-player.js:188]: This should probably use `registerAction`. `registerDefaultAction` would call `destroy` when using only the element's id without an action name like `<button on="tap:my-brid-player">`.

```suggestion
    this.registerAction(BridActions.DESTROY, () => this.destroy());
```

**Comment by grajzer** [comment]
Hi @alanorozco thanks for taking a look. Here I'll try to explain what I mean by actions/events and please tell me if I used the wrong terminology or did something wrong.

BridEvents (adImpression, adError, adSkip, adComplete) are dispatched from AmpBridPlayer so users can listen to them:

<amp-brid-player
   id="myVideo"
   ...
   on="adError:...."
>

BridActions (destroy) are used so you can invoke them:

<button on="tap:myVideo.destroy">Destroy player</button>

**Pull request closed without merging**